vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher_test.go

```diff
@@ -340,8 +353,31 @@ func TestControllerFetcher(t *testing.T) {
 				Name: "iCanScale", Kind: "Scale", Namespace: "test-namesapce"}, ApiVersion: "Foo/Foo"}, // Parent supports scale subresource"
 			expectedError: nil,
 		},
+		{
+			name: "pod, parent is node",
+			objects: []runtime.Object{&appsv1.Deployment{
+				TypeMeta: metav1.TypeMeta{
+					Kind: "Deployment",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-deployment",
+					Namespace: "test-namesapce",
+					// Parent that support scale subresource and is not well known
```

Comment:
nit: s/support/supports/

Reply:
I replaced the comment with a more accurate one